systemtest/src/test/java/io/strimzi/systemtest/specific/ClusterOperationST.java

```diff
@@ -32,16 +32,19 @@
 public class ClusterOperationST extends AbstractST {
 
     private static final Logger LOGGER = LogManager.getLogger(ClusterOperationST.class);
+    private KafkaBasicClientResource kafkaBasicClientResource;
```

Comment:
What is the value of having this at the class level? Looks like you use it only within a single loop.